src/utils/parser.py

```diff
@@ -85,6 +85,14 @@
     "kernelBusyCycles": "ROUND(AVG((((End_Timestamp - Start_Timestamp) / 1000) * $max_sclk)), 0)",
 }
 
+# Section based configuration for profiling and anlysis
+section_config = {
+    "SOL": {"id": 200, "source": "0200_system-speed-of-light.yaml"},
+    "MEMCHART": {"id": 300, "source": "0300_mem_chart.yaml"},
+    "WAVEFRONT": {"id": 700, "source": "0700_wavefront-launch.yaml"},
+    "INSTMIX": {"id": 1000, "source": "1000_compute-unit-instruction-mix.yaml"},
```

Comment:
As discussed today, would be better to include FLOPs throughput metrics with this option, and maybe change name to `COMPUTE`?